### PR TITLE
Fix/TEC-3635 - Make default limit by number of events instead of by date range

### PR DESCRIPTION
**Pull request by codingmusician**
JIRA Ticket: [TEC-3635](https://moderntribe.atlassian.net/browse/TEC-3635)

* Having the default import limit setting be by number instead of date range is a better UX for our customers

### CODE REVIEW

File: src/admin-views/aggregator/origins/limit.php [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?php
 /** @var \Tribe__Events__Aggregator__Settings $settings */
 $settings             = tribe( 'events-aggregator.settings' );
-$global_limit_type    = tribe_get_option( 'tribe_aggregator_default_import_limit_type', 'range' );
+$global_limit_type    = tribe_get_option( 'tribe_aggregator_default_import_limit_type', 'count' );
```
Comment (tr1b0t) [inline src/admin-views/aggregator/origins/limit.php:4]: :warning: Equals sign not aligned with surrounding assignments; expected 1 space but found 4 spaces

<sub>hash: 2565573280bb0a8275c414bb0e9571f4</sub>

**Review by tr1b0t — COMMENTED** [review]
:warning: 1 warning(s)

**Review by mitogh — COMMENTED** [review]
Looks good make sure the bot comments are addressed as well with the git conflicts.

File: src/admin-views/aggregator/settings.php [hunk 2]
```diff
@@ -174,7 +174,7 @@
 			'tooltip'         => esc_html__( 'Limit the number of imported events by number, date range, or not at all; on slower websites this may impact the success of imports. Selecting a shorter time period or a smaller number of events may improve results.', 'the-events-calendar' ),
 			'size'            => 'medium',
 			'validation_type' => 'options',
-			'default'         => 'range',
+			'default' => 'count',
```
Comment (tr1b0t) [inline src/admin-views/aggregator/settings.php:177]: :warning: Array double arrow not aligned correctly; expected 9 space(s) between "'default'" and double arrow, but found 1.

<sub>hash: 4f3bddbe80477b73b7d770289e3efa4f</sub>

**Review by tr1b0t — COMMENTED** [review]
:warning: 1 warning(s)

**Review by mitogh — DISMISSED** [review]

**Comment by mitogh** [comment]
Can you make sure the additional changes from this PR are intentional ?

Thanks @codingmusician

**Pull request closed without merging**